tests/v2/validation/charts/gatekeeper_test.go

```diff
@@ -46,6 +49,8 @@ func (g *GateKeeperTestSuite) SetupSuite() {
 	clusterID, err := clusters.GetClusterIDByName(client, clusterName)
 	require.NoError(g.T(), err)
 
+	logrus.Infof("debugging what is the clusterName %v", clusterID)
```

Comment:
Is this supposed to be here or was this for debugging purposes?

Reply:
removed it